src/blop/integrations/ax/helpers.py

```diff
@@ -59,6 +60,35 @@ def create_bluesky_evaluator(
     """
     plan_function = plan or list_scan
 
+    def convert_to_dictonary(db) -> dict[str, list[Any]]:
+        """
+        Converts the data that arrives from either databroker as a pd.Dataframe or through
+        tiled as a xr.DataArray into a dictonary
+
+        Parameters
+        ----------
+        db : databroker or tiled
+            The databroker or tiled instance
+        """
+        dictonary = {}
+        if isinstance(db, tiled.client.container.Container):
+            tiled_data = db["streams", "primary"].read()
+            for var_name, data_array in tiled_data.data_vars.items():
+                dictonary[var_name] = data_array.values.flatten().tolist()
+            tiled_image = db["streams", "primary", "bl_det_image"].read().astype(float)
+            dictonary["bl_det_image"] = tiled_image
+            return dictonary
+
+        # if it is a databroker instance
+        elif isinstance(db, list):
+            data = db[0].table(fill=True)
+            for i in data:
+                dictonary[i] = data[i].to_list()
+            return dictonary
+
+        else:
+            raise ValueError("Unknown data source.")
+
```

Comment:
Please use the `data_access.py` interface here instead of copying the same code again in this file.